docker/README.md

```diff
@@ -69,8 +69,8 @@ docker rm -f mms
 ## Available pre-built continers
 We have 5 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
 1. *latest*: This will be the same as `mxnet-model-server:1.0.0-mxnet-cpu`. This tag will be available after an official release.
```

Comment:
This line need to be changed as well

Reply:
This is currently pointing to 1.0.0 as this wasn't built for 1.0.1. I think we should leave this for now and update this for 1.0.2 release.